src/components/featureHighlight/index.js

```diff
@@ -136,56 +145,82 @@ class FeatureHighlight extends BaseComponent {
     return findNodeHandle(target);
   }
 
+  animate(toValue) {
+    Animated.timing(                  // Animate over time
+      this.state.fadeAnim,            // The animated value to drive
+      {
+        toValue,                      // Animate to value
+        duration: toValue === 0 ? 0 : 100,   // Make it take a while
+      },
+    ).start();                        // Starts the animation
+  }
+
   setTargetPosition(props = this.props) {
     if (props.getTarget !== undefined) {
       const target = props.getTarget();
 
       const node = this.findTargetNode(target);
       this.setState({node});
-
       if (target) {
         setTimeout(() => {
           target.measureInWindow((x, y, width, height) => {
-            this.setState({
-              targetPosition: {left: x, top: y, width, height},
-            });
+            this.targetPosition = {left: x, top: y, width, height};
+            this.setContentPosition();
           });
         }, 0);
       }
     } else {
       const frame = props.highlightFrame;
       if (frame) {
-        this.setState({
-          targetPosition: {left: frame.x, top: frame.y, width: frame.width, height: frame.height},
-        });
+        this.targetPosition = {left: frame.x, top: frame.y, width: frame.width, height: frame.height};
```

Comment:
if we're setting `this.targetPosition` every time we're calling `this.setContentPosition()`, let's just move the assignment inside the function, it has the right name anyway.